phoenix-core/src/it/java/org/apache/phoenix/end2end/CostBasedDecisionIT.java

```diff
@@ -0,0 +1,176 @@
+package org.apache.phoenix.end2end;
+
+import static org.apache.phoenix.util.TestUtil.TEST_PROPERTIES;
+import static org.junit.Assert.assertTrue;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.Map;
+import java.util.Properties;
+
+import org.apache.phoenix.query.BaseTest;
+import org.apache.phoenix.query.QueryServices;
+import org.apache.phoenix.util.PropertiesUtil;
+import org.apache.phoenix.util.QueryUtil;
+import org.apache.phoenix.util.ReadOnlyProps;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import com.google.common.collect.Maps;
+
+public class CostBasedDecisionIT extends BaseTest {
```

Comment:
Derive from BaseUniqueNamesOwnClusterIT instead so that it runs when mvn verify is run (it's based on the annotation which this won't have)

Reply:
Thanks for pointing this out, @JamesRTaylor! Changed in my latest CI as suggested.